vowpalwabbit/parse_args.cc

```diff
@@ -1624,10 +1637,17 @@ vw* initialize(std::unique_ptr<options_i, options_deleter_type> options, io_buf*
   }
 }
 
-vw* initialize(std::string s, io_buf* model, bool skip_model_load, trace_message_t trace_listener, void* trace_context)
+vw* initialize(std::unique_ptr<options_i, options_deleter_type> options, io_buf* model, bool skip_model_load,
```

Comment:
most of these fns have been shuffled around, see vw.h for a diff on what actually changed from the public side.